lms/static/js/Markdown.Editor.js

```diff
@@ -63,102 +62,97 @@
     // - getConverter() returns the markdown converter object that was passed to the constructor
     // - run() actually starts the editor; should be called after all necessary plugins are registered. Calling this more than once is a no-op.
     // - refreshPreview() forces the preview to be updated. This method is only available after run() was called.
-    Markdown.Editor = function (markdownConverter, idPostfix, help, imageUploadHandler) {
-
-        idPostfix = idPostfix || "";
+    Markdown.Editor = function(markdownConverter, idPostfix, help, imageUploadHandler) {
+        idPostfix = idPostfix || '';
 
         var hooks = this.hooks = new Markdown.HookCollection();
-        hooks.addNoop("onPreviewPush");       // called with no arguments after the preview has been refreshed
-        hooks.addNoop("postBlockquoteCreation"); // called with the user's selection *after* the blockquote was created; should return the actual to-be-inserted text
-        hooks.addFalse("insertImageDialog");     /* called with one parameter: a callback to be called with the URL of the image. If the application creates
+        hooks.addNoop('onPreviewPush');       // called with no arguments after the preview has been refreshed
+        hooks.addNoop('postBlockquoteCreation'); // called with the user's selection *after* the blockquote was created; should return the actual to-be-inserted text
+        hooks.addFalse('insertImageDialog');     /* called with one parameter: a callback to be called with the URL of the image. If the application creates
                                                   * its own image insertion dialog, this hook should return true, and the callback should be called with the chosen
                                                   * image url (or null if the user cancelled). If this hook returns false, the default dialog will be used.
                                                   */
         this.util = util;
 
-        this.getConverter = function () { return markdownConverter; }
+        this.getConverter = function() { return markdownConverter; };

```

Comment:
I noticed that the changes added ;s to the end of a couple lines in this file. 
Will that change the functionality, or is it more syntactically correct?